examples/timer/src/main.rs

```diff
@@ -4,16 +4,20 @@ use yew::services::{ConsoleService, Task, TimeoutService};
 use yew::{html, Callback, Component, ComponentLink, Html, ShouldRender};
 
 pub enum Msg {
+    StartClock,
     StartTimeout,
     StartInterval,
     Cancel,
     Done,
     Tick,
+    Tock,
```

Comment:
As much as I appreciate the pun, should we go for something more descriptive like `UpdateTime`?